freedata_gui/src/components/dynamic_components.vue

```diff
@@ -449,7 +518,7 @@ onMounted(() => {
    <!-- <i class="bi bi-grip-vertical h5">      </i>-->
 
   <span class="me-4 text-white" style="writing-mode: vertical-rl; transform: rotate(180deg);">
```

Comment:
the `me-4` seems to be the key. If we remove it, we have a regular view on Firefox, but the text is hidden on Safari.